src/common.js

```diff
@@ -12,6 +12,7 @@ exports.config = config;
 var state = {
   error: null,
   currentCmd: 'shell.js',
+  previousDir: null,
```

Comment:
In bash, `OLDPWD` (the env variable that holds the previous directory) is initially unset. So running `$ cd -` in a new bash shell is an error, which is why I'm initializing to null and checking the value.

Either way, most people aren't going to run `cd('-')` without first having been in some directory, so it's probably not a big concern to have either behavior.